src/start.sh

```diff
@@ -232,13 +238,22 @@ PID=$(cat "${PID_FILE}" 2>/dev/null);
 if [ -z "${PID}" ]; then
     echo "ID:  ${HID}"
     [ ${VERBOSE} ] && echo "PID file for this Hazelcast member: $PID_FILE"
-    [ ${VERBOSE} ] && echo "Permanent logfile for this Hazelcast member: $LOG_FILE"
+    [ ${VERBOSE} -a ! ${CONF_FOREGROUND} ] && echo "Permanent logfile for this Hazelcast member: $LOG_FILE"
     HZ_CMD="${RUN_JAVA} -server ${JAVA_OPTS} com.hazelcast.core.server.StartServer"
     [ ${VERBOSE} ] && echo "Command line: $HZ_CMD"
-    nohup ${HZ_CMD} >>"${LOG_FILE}" 2>&1 &
+    if [ ${CONF_FOREGROUND} ] ; then
+        set -m
+        ${HZ_CMD} &
```

Comment:
Don't you want to do both: log to the console and to the file?

Reply:
Good question really. The use case I had in mind was that you'd use `docker logs` while running with `-fg` in a Docker container, so there'd be no real need to create a log file in the container itself. Do you think it's useful to have both?

Reply:
I'm going to merge this. Let's implement the file+console logging in another PR.